### ISSUE TITLE
Missing Texture

### ISSUE THREAD

**Issue by ardissaps** [CebbysMods/slabref-mod#17]
![image](https://user-images.githubusercontent.com/82106625/173335372-0fa08b01-ad35-4822-a3ac-e9504aecd115.png)

**Comment by CebbyS** [comment]
Hi, @ardissaps ! Can you explain how did you get this exception? What was the mod you were using? Did you use the set block command? I would like to know the steps to replicate this issue?

Thanks and regards!

**Comment by ardissaps** [comment]
I had mod named Kleeslab that used for destroying half slab of similiar so that not full block not all destroyed, also MoreCulling from fxmorin.

**Comment by ardissaps** [comment]
![sdasdadasdad](https://user-images.githubusercontent.com/82106625/173463467-6f4241b3-1a5b-4478-9e16-944ca4776d0b.jpg)
Which could be causing?

**Comment by ardissaps** [comment]
Ignore the "Vanilla Parts" mod, as it is my temporal solution that was I use before using Refabricated Slab, I prefer your mod because compatible with MoreCulling.

**Comment by CebbyS** [comment]
Sorry for the late response...
And I might not have been clear.

So what I would like have received are (Steps on how to reproduce the issue):
1. Place slab.
2. Place another slab on the top or bottom of the first slab.
3. Placement was successful.
4. Break the block using kleeslab mod.
5. On block broken, block with no textures appears.

// This is my understanding of what you wrote in the response, can you please confirm if this is how to reproduce this bug?

And another thing that could help is logs from the console. It would be nice if you could provide me the logs on https://pastebin.com/ when this bug happens? (Start Minecraft with console logs, open the world, reproduce the bug, copy logs and share them on Pastebin)

But in the end, the best way to understand the issue and help you would be direct messaging. Maybe we can exchange messages in Discord? You can find me on Discord with cebbyS#2310

I will be online at about 10:00 (GMT+3)

**Comment by CebbyS** [comment]
Hi, I have identified the issue! I am working on a patch to fix this! Sorry for the inconvenience!

Release is in progress, but the mod still will not work with KleeSlabs, I have added a comment to the thread you opened on the github page, so that it would be possible for me to create a compatibility patch for KleeSlabs. 

For now, the Refabricated Slabs should work fine, but you will not be able just to break half of the double slabs of, breaking double slab will break the whole block.

![image](https://user-images.githubusercontent.com/56877579/174054229-e898d74c-865e-4950-9139-75a73d8dd0a6.png)
